models/staging/tpcds_core/_tpcds_models.yml

```diff
@@ -60,7 +60,7 @@ models:
           - not_null
 
   - name: stg_tpcds_core__income_band
-    description: staging model for the income_band table from the core raw schema
+    description: staging model for the income_band table from the core raw schema!
```

Comment:
This is definitely a more _exciting_ description.

Not against it, but curious to hear the rationale for the exclamation!